src/components/Comment/Comment.jsx

```diff
@@ -0,0 +1,149 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Textarea from 'react-textarea-autosize';
+import { PropTypes } from 'prop-types';
+import {
+  handleCommentInput,
+  submitComment,
+  deleteComment,
+  updateComment,
+  setUpdateCommentBody,
+} from '../../redux/actions/commentAction';
+import CommentRender from './CommentRender';
+
+export class Comment extends Component {
+  addComment = () => {
+    const { body, articleSlug, onSubmitComment } = this.props;
+    onSubmitComment(body, articleSlug);
+  }
+
+  onChange = (e) => {
+    const { onCommentInput } = this.props;
+    onCommentInput({ body: e.target.value });
+    e.preventDefault();
+  }
+
+  displayComments = () => {
+    const {
+      commentList,
+      currentUser,
+      onDeleteComment,
+      articleSlug,
+      onUpdateCommentInput,
+    } = this.props;
+    if (commentList.length) {
+      const newList = commentList.sort((a, b) => new Date(b.createdAt) - new Date(a.createdAt));
+      return (
+        newList.map(comment => (
+          <CommentRender
+            commentList={commentList}
+            currentUser={currentUser}
+            onDeleteComment={onDeleteComment}
+            articleSlug={articleSlug}
+            comment={comment}
+            key={comment.id}
+            enterPress={this.onEnterPress}
+            updateComment={this.onEditComment}
+            inputHandler={onUpdateCommentInput}
+          />
+        ))
+      );
+    }
+    return '';
```

Comment:
Instead of returning an empty string, why not return a place holder? To give that `visual feedback` to the user.

We also always start with the negative rather than the positive, hence https://softwareengineering.stackexchange.com/questions/254002/checking-negative-of-a-condition

Therefore we should structure the code in such a manner:
```
if(!commentList.length) {
   return 'placeholder';
} 
return (blah blah blah your component);
```

Reply:
Actually, I don't know what a placeholder is but let me try and this the result